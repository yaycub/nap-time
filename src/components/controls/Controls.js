import React from 'react';
import PropTypes from 'prop-types';
import styles from './Controls.css';

const Controls = ({ children }) => (
  <section className={styles.Controls}>
    {children}
  </section>
);

Controls.propTypes = {
  children: PropTypes.node
};

export default Controls;
