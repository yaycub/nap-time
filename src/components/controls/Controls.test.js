import React from 'react';
import { shallow } from 'enzyme';
import Controls from './Controls';

describe('Controls component', () => {
  it('should render Controls', () => {
    const wrapper = shallow(<Controls />);
    expect(wrapper).toMatchSnapshot();
  });
});
