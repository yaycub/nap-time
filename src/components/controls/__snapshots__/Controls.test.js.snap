// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controls component renders Controls 1`] = `
<section
  className="Controls"
/>
`;
